### PR TITLE
add new leaf tag from Sugar to format a number to fixed decimals

### PR DESCRIPTION
**Pull request by themitvp**
what do you think of the naming? Should we maybe instead call it `number:format` or `number:decimals` so we in the future can add more types for example `number:currency` or something ?

### CODE REVIEW

**Comment by steffendsommer** [comment]
Sorry for the delay. How about just `number`? The number of decimals is then the input. Let's get it building and then if you agree with the name then I think we're good to merge.

**Comment by rasmusebbesen** [comment]
Should we cancel this PR?

**Comment by steffendsommer** [comment]
The tag is still in Sugar and this is quite convenient and common, so I vote for renaming it and merging it.

File: Sources/AdminPanel/Providers/AdminPanelProvider.swift [hunk 1]
```diff
@@ -139,7 +139,8 @@ public final class AdminPanelProvider<U: AdminPanelUserType>: Provider {
             "adminPanel:user": CurrentUserTag<U>(),
             "adminPanel:user:requireRole": RequireRoleTag<U>(),
             "adminPanel:user:hasRequiredRole": HasRequiredRole<U>(),
-            "submissions:WYSIWYG": InputTag(templatePath: config.tagTemplatePaths.wysiwygField)
+            "submissions:WYSIWYG": InputTag(templatePath: config.tagTemplatePaths.wysiwygField),
+            "numberformat": NumberFormatTag()
```
Comment (siemensikkema) [inline Sources/AdminPanel/Providers/AdminPanelProvider.swift]:
```suggestion
            "number": NumberFormatTag()
```

**Comment by siemensikkema** [comment]
depends on https://github.com/nodes-vapor/sugar/pull/99

File: Sources/AdminPanel/Providers/AdminPanelProvider.swift [hunk 2]
```diff
@@ -139,6 +139,7 @@ public extension LeafTagConfig {
             "adminPanel:user": CurrentUserTag<U>(),
             "adminPanel:user:requireRole": RequireRoleTag<U>(),
             "adminPanel:user:hasRequiredRole": HasRequiredRole<U>(),
+            "numberformat": NumberFormatTag(),
```
Comment (siemensikkema) [inline Sources/AdminPanel/Providers/AdminPanelProvider.swift]:
```suggestion
            "number": NumberFormatTag(),
```

**Comment by siemensikkema** [comment]
@steffendsommer ping :)

**Pull request merged**